pkg/virt-api/webhooks/mutating-webhook/preset.go

```diff
@@ -65,7 +65,7 @@ func filterPresets(list []kubev1.VirtualMachineInstancePreset, vmi *kubev1.Virtu
 	return matchingPresets, nil
 }
 
-func checkMergeConflicts(presetSpec *kubev1.DomainSpec, vmiSpec *kubev1.DomainSpec) error {
+func checkMergeConflicts(presetSpec *kubev1.PresetDomainSpec, vmiSpec *kubev1.PresetDomainSpec) error {
```

Comment:
is checkMergerConflicts used anymore? Can it be removed?

Reply:
It is still used actually--to ensure VM presets can't conflict with each other.